bin/pycbc_make_skymap

```diff
@@ -542,6 +551,9 @@ if __name__ == '__main__':
     parser.add_argument('--fake-strain-seed', type=int,
                         action=MultiDetOptionAction, metavar="IFO:FAKE_STRAIN_SEED",
                         help="The set of fake-strains-seeds for each detector")
+    parser.add_argument('--fudge',
```

Comment:
Two comments on these lines (unrelated to @spxiwh's comment):
* The argument takes a float value, so use `type=float`.
* Remove `default=None` since `None` is already the default value.

Reply:
> The argument takes a float value, so use type=float.

Nevermind, it actually makes sense to keep it as a string, since we just pass it verbatim to BAYESTAR.

Reply:
@spxiwh  I changed the name of the argument for `--rescale-loglikelihood`, to be consistent with Bayestar.

@titodalcanton 
Seems ok to keep a string as it is only used as it in the script. It also avoid round issues when converting the float into a string.
I removed the `None` default value.

Reply:
@spxiwh the BAYESTAR help string says

> Rescale log likelihood by the square of this factor…

So it really is an SNR scale factor (perhaps modulo a factor 2). So I think @PAduverne's help message is fine.